communication/marketing-team/team-resources.md

```diff
@@ -0,0 +1,21 @@
+# Resources for the team
+[Entire community repo]  
+The future home of the [contributor site], [project board]  
+[surveys]  
+[Communication]  
+[style guide]  
+[Org structure] and [Community Groups List]
+[Values] and [principles] of the project  
+
+
+
+[contributor site]: https://github.com/kubernetes-sigs/contributor-site
+[project board]: https://github.com/orgs/kubernetes-sigs/projects/3
+[surveys]: https://github.com/kubernetes/community/tree/master/sig-contributor-experience
+[Communication]: https://github.com/kubernetes/community/tree/master/communication
+[style guide]: https://git.k8s.io/community/contributors/guide/style-guide.md
+[Org structure]: https://github.com/kubernetes/community/blob/master/governance.md
+[Community Groups List]: https://github.com/kubernetes/community/blob/master/sig-list.md
+[Values]: https://github.com/kubernetes/community/blob/master/values.md
+[principles]: https://github.com/kubernetes/community/blob/master/governance.md#principles
+[Entire community repo]: https://github.com/kubernetes/community
```

Comment:
Similar to the earlier comment, these should all be relative links to their locations in the community repo.